apollo-ast/src/main/resources/builtins.graphqls

```diff
@@ -135,5 +135,5 @@ directive @deprecated(
 
 directive @defer(
   label: String
-  if: Boolean
+  if: Boolean! = true
```

Comment:
Not related to this PR, but noticed this was not up to date with the latest spec.